hail/python/hailtop/batch/backend.py

```diff
@@ -64,7 +82,7 @@ def _run(self, batch, dry_run, verbose, delete_scratch_on_exit, **backend_kwargs
 
     @property
     @abc.abstractmethod
-    def _fs(self) -> AsyncFS:
+    def _fs(self) -> "RouterAsyncFS":
```

Comment:
I think quoting types is only necessary when that type is only defined later in the import process. We import `RouterAsyncFS` at the top of this file, does this have to be quoted?

Reply:
this was left over from when i was trying to fix the docs build, removed the quotes